src/app/transloco-loader.ts

```diff
@@ -0,0 +1,12 @@
+import {inject, Injectable} from '@angular/core';
+import {Translation, TranslocoLoader} from '@jsverse/transloco';
+import {HttpClient} from '@angular/common/http';
+
+@Injectable({providedIn: 'root'})
+export class TranslocoHttpLoader implements TranslocoLoader {
+  private http = inject(HttpClient);
+
+  public getTranslation(lang: string) {
```

Comment:
Regarding return types, I would suggest using a linter rule to enforce them rather than hoping that people will adhere to the convention (that they might not know). I would also argue that it is a good idea to add the void return type to make it easier to spot unintentional return type changes. Without return type TypeScript happily infers the type.

Reply:
That sounds perfect 👌
I'm also happy with adding `void` for methods that shouldn't return anything. It makes the intention more explicit and leaves less room for errors.

Reply:
I would add the ESlint config in a separate PR